src/app/index.tsx

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
-import { SafeAreaView, StatusBar } from 'react-native';
-import { WHITE } from '@constants/colors';
 import HomeScreen from '@screens/HomeScreen';
+import DetailScreen from '@screens/DetailScreen';
+import 'react-native-gesture-handler';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import { DETAILSCREEN, HOMESCREEN } from '@constants/screenNames';
```

Comment:
Use snake_case for constants: `DETAIL_SCREEN ` and `HOME_SCREEN`:

`import { DETAIL_SCREEN, HOME_SCREEN } from '@constants/routes';`